packages/server-wallet/src/protocols/__test__/defund-channel.test.ts

```diff
@@ -187,26 +187,6 @@ it('should fail the objective when the channel does not exist', async () => {
   expect(reloadedObjective.status).toEqual('failed');
 });
 
-it('should fail when using non-direct funding', async () => {
-  const chainService = new MockChainService();
-  const channelDefunder = ChannelDefunder.create(store, chainService, logger, timingMetrics);
-
-  // Create a channel with fake funding strategy
-  const c = channel({fundingStrategy: 'Fake'});
```

Comment:
I think we should keep this test but use ledger funding:
 ` const c = channel({fundingStrategy: 'Ledger'});`